docs/app/Component/ComponentProps.js

```diff
@@ -14,13 +14,27 @@ export default class ComponentProps extends Component {
     props: PropTypes.object
   };
 
+  nameRenderer(item) {
+    let required = item.required && <span className='ui mini red circular label'>required</span>;
+    return <div>{item.name} {required}</div>;
+  }
+
+  defaultValueRenderer(item) {
+    let defaultValue = _.get(item, 'defaultValue.value');
+    let defaultIsComputed = <span className='ui mini gray circular label'>computed</span>;
+
+    return (
+      <div>
+        {defaultValue} {_.get(item, 'defaultValue.computed') && defaultIsComputed}
+      </div>
+    );
+  }
```

Comment:
The Default Value column includes a `computed` label if the prop value is computed from a function.